app/graph/relay_on_rails_schema.rb

```diff
@@ -1,7 +1,13 @@
 class RelayOnRailsSchema < GraphQL::Schema
   query QueryType
   mutation MutationType
+
   use GraphQL::Batch
+
+  # Opt in to the new runtime (default in future graphql-ruby versions)
```

Comment:
TODO: Figure out what we need to do here, and whether we want to opt in to the execution interpreter